neqo-crypto/src/aead.rs

```diff
@@ -76,15 +76,14 @@ impl Aead {
         let prefix_str = prefix.into();
         let p = prefix_str.as_bytes();
         let mut ctx: *mut ssl::SSLAeadContext = null_mut();
-        let rv = SSL_MakeAead(
+        SSL_MakeAead(
             version,
             cipher,
             secret,
             p.as_ptr() as *const i8,
-            p.len() as u32,
+            p.len().try_into().unwrap(),
```

Comment:
prefer tryfrom? e.g.  `u32::try_from(p.len()).unwrap()`. More explicit about target type.

Reply:
I think that I might prefer try_from where possible.  That doesn't seem to be idiomatic and I don't want to just hit this one instance when we do that.